### DISCUSSION TITLE
API is not returning correct property names on product_data field

### DISCUSSION THREAD

**Discussion by jakeleventhal** [etsy/open-api#1334] [Ideas]
![image](https://github.com/user-attachments/assets/a7711645-abc6-4a4b-a8e3-04f23fe02978)
The `product_data` field gives us "Custom Property" always for all properties instead of the real name

**Comment by turboturtleD** [comment]
Can you share the listing ID for this item?

**Reply by jakeleventhal** [reply]
pretty much any listing. here is one of them (1825696609). this issue doesnt show up in `getListingsByShop`. This problem is when you look at the `product_data` field in `getShopReceipts`
